Adafruit_MPL115A2.cpp

```diff
@@ -80,15 +40,16 @@ void Adafruit_MPL115A2::readCoefficients() {
   int16_t b2coeff;
   int16_t c12coeff;
 
-  _wire->beginTransmission(_i2caddr);
-  i2cwrite(_wire, (uint8_t)MPL115A2_REGISTER_A0_COEFF_MSB);
-  _wire->endTransmission();
+  uint8_t cmd;
+  uint8_t buffer[8];
```

Comment:
For all these i2c xfers - how about adding `buffer` as a member and then using it for both outbound and inbound:
```cpp
buffer[0] = MPL115A2_REGISTER_A0_COEFF_MSB;
_i2c_dev->write_then_read(buffer, 1, buffer, 8);
```
etc.

Size it for max needed, which looks like maybe `8`?

Reply:
this works as-is, & dont really want to do anymore work on this library...even if its not as elegant as it could be!